maintainers/maintainer-list.nix

```diff
@@ -1220,6 +1220,16 @@
     github = "benneti";
     githubId = 11725645;
   };
+  bertof = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Ok